frontend/src/scenes/plugins/tabs/installed/InstalledTab.tsx

```diff
@@ -69,7 +70,9 @@ export function InstalledTab(): JSX.Element {
     } = useActions(pluginsLogic)
 
     const upgradeButton =
-        user?.plugin_access.install && hasNonSourcePlugins ? (
+        user?.organization &&
+        user.organization.plugins_access_level === PluginsAccessLevel.Root &&
+        hasNonSourcePlugins ? (
```

Comment:
If you have install access, you should still be able to upgrade your own plugins. The individual upgrade buttons still show now...